Good-Bids/goodbids
# [#495] Misc Fixes

# Summary

This PR generates a new Navigation post type if one doesn't exist in order to set up the navigation during the site init. This needs to be tested, but I don't think it will show as updated until you either (1) visit WP Admin, or (2) visit the Site Editor, but see if you see the updated navigation by visiting the front-end after you create the site.

This PR also:

* Suppresses a few errors that were showing up in the logs during site creation, but there are a ton more I couldn't figure out (they're not affecting anything, it's just annoying).
* Updates Accessibility Checker, WooCommerce Services, and WooCommerce Stripe Gateway to the latest version.
* Removes the wp-cli dev dependency.
* Adds a new `get_view` method to avoid having to do `ob_start()` and `ob_get_contents()` for view files.
* Adjusts the priorities of some of the actions, allowing for many of the things to occur during site initialization.
* As of WP VIP 6.1, WP VIP no longer requires usage of `wpcom_vip_get_page_by_path()`, so that method has been simplified to the native WP Function.
* Fixes a bug when SVG Support doesn't have any options yet
* Retains all the $_POST data when creating new Nonprofit site but the data fails validation.
* Fixes a bug with the `goodbids_nonprofit_verified` hook that was firing anytime `verification` was updated (instead of only when it is set to `true`).

## Issues

* #495 

## Testing Instructions

1. Create a new Nonprofit site
2. Try to enter an invalid EIN on the Verification step.
3. Make sure all the fields retain their information
